src/types/ZoomCloudRecording.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Zoom cloud recording object
+ * @author Gabe Abrams
+ */
+type ZoomCloudRecording = {
+  
+};
```

Comment:
This surprises me that it's an empty object

Reply:
Yeah that's cause it shouldn't be. Somehow that didn't get pushed. I'll check in and fix this.